.travis.yml

```diff
@@ -0,0 +1,24 @@
+---
+language: python
+python:
+  - "3.5"
+  - "3.6"
+matrix:
```

Comment:
An alternative to the `matrix` config item approach is the newer `jobs` config.

* https://docs.travis-ci.com/user/build-stages/#how-to-define-build-stages

Might be easier to understand that option, for configuring multiple jobs.

Reply:
I've got that approach working over in Pyppyn: https://github.com/YakDriver/pyppyn/blob/dev/.travis.yml